packages/tempus/src/Tempus_StepperBDF2_impl.hpp

```diff
@@ -20,6 +20,7 @@ namespace Tempus {
 template<class Scalar>
 StepperBDF2<Scalar>::StepperBDF2()
 {
+  this->setStepperName(        "BDF2");
   this->setStepperType(        "BDF2");
```

Comment:
We need to document the distinction between "`StepperName` and `StepperType`.

Reply:
With latest commit, added documentation on stepperName and stepperType.